plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml

```diff
@@ -22,7 +22,18 @@ spec:
         type: string
         description: Unique name of the component
       description:
-        title:  Description
+        title: Description
         type: string
         description: Description of the component
-
+---
```

Comment:
Remove it from this file and move to a separate example in the catalog-backend? Or you can create `mock-data` script in the github-actions plugin' package.json and store this entity in the github-actions folder.

Also, this brings an interesting question to the table - if we are to allow scaffolder to create new repo with github actions out-of-the-box, how do we make sure it's going to be picked by the github-actions plugin. Like, do we hardcode `backstage.io/github-actions-id: 'spotify/backstage'` in the scaffolder sample-template. Or even import the `backstage.io/github-actions-id` as const from @backstage/plugin-github-actions? @Nek @benjdlambert  wdyt?

Reply:
On the first one, yes, definitely. I think I'll go with a separate mock-data.

Reply:
So i might have misunderstood, so bear with me...

Is it strictly necessary to store it as an annotation? if you have the source repo location already does it need to be added as an annotation?.

My thinking is that when you create a repo, and it's added to the service catalog, you should already know where the repo lives from the service definition that lives in the service catalog. Can you not query using the source URL or is the annotation there to solve a problem here?

What's the extra ID for?

Reply:
@benjdlambert but is it always the same repo? I can imagine a case when you have, let's say, a submodule which is continuously integrated only in the context of the parent project.

Reply:
Yeah ok maybe that's a good enough use case